src/components/atoms/icons.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+
+export const IconRight = () => (
```

Comment:
All the SVGs should have the values you fixed as props. width, height, viewBox, fill should all be dynamic so we can set them differently if needs be.

Reply:
I doubt there will be a need to change their sizes but I'll work on that too